tests/DCPS/MetaStruct/MetaStructTest.cpp

```diff
@@ -59,6 +60,30 @@ int check(const T& lhs, const T& rhs, const char* name)
   return 0;
 }
 
+template<typename T, typename T2>
+int check(const T& lhs, const T& rhs, const char* name, ACE_Message_Block* amb, Value::Type type,
+          const MetaStruct& ms, T2 Value::*ptrmbr)
+{
+  if (checkVal(lhs, rhs, name)) return 1;
+
+  Message_Block_Ptr mb (amb->duplicate());
+  const Encoding encoding(OpenDDS::DCPS::Encoding::KIND_UNALIGNED_CDR, OpenDDS::DCPS::ENDIAN_NATIVE);
+  OpenDDS::DCPS::Serializer ser(mb.get(), encoding);
```

Comment:
```suggestion
  const Encoding encoding(Encoding::KIND_UNALIGNED_CDR);
  Serializer ser(mb.get(), encoding);
```

Native is the default argument and there's a `using namespace OpenDDS::DCPS` at the top.